src/app/review/agree/page.tsx

```diff
@@ -0,0 +1,96 @@
+"use client";
+
+import * as style from "./style.css";
+import { useState } from "react";
+import { useAgreeReviewMutation } from "@app/api/review/agree/hook";
+import { TITLE, CHECKBOX_TEXTS } from "@app/review/agree/constant";
+import Image from "next/image";
+
+import { IcLeftarrow } from "@asset/svg";
+import reviewNoticeFrame from "@asset/image/reviewNoticeFrame.png";
+import danger from "@asset/image/danger.png";
+import { IcCheckbox } from "@asset/svg";
+
+import HeaderNav from "@common/component/HeaderNav/HeaderNav";
+import Divider from "@common/component/Divider/Divider";
+import { Button } from "@common/component/Button";
+
```

Comment:
p2) 전체적으로 매직넘버를 많이 사용하고 있는 것 같은데, 지양하면 좋겠어요! 
아래의 코드를 추가해서 사용하면 좋을 것 같아요.

```suggestion
const ALL_CHECKBOX_IDX = 0;
const SUB_CHECKBOX_START_IDX = 1;
```

Reply:
오.. 생각하지 못했던 부분인데 피드백 감사합니다!!
반영하겠습니다..!

Reply:
반영완료했습니다!